jekyll/_cci2/customizing-ios.md

```diff
@@ -0,0 +1,429 @@
+https://rink.hockeyapp.net/manage/auth_tokens). Your token will need at
+least upload permission to upload new builds to Hockey App. 
+
+2. Give your
+new API token a name specific to CircleCI such as "CircleCI
+Distribution". 
+
+3. Copy the token, and log into CircleCI and go to the
+Project Settings page for your app. 
+
+4. Create a new Environment Variable with
+the name `HOCKEY_APP_TOKEN` and paste the token as the value. You can now
+access this token during the build.
+
+5. Modify the deployment section of your `config.yml` as follows:
+
+```
+    steps:
+      - deploy:
+          name: hockey app deploy
+          command: |
+            - gym
+            - ipa distribute:hockeyapp
+                --file             /Users/distiller/<yourprojectname>/<yourappname>.ipa
+                --token            "$HOCKEY_APP_TOKEN"
+                --notes            "CircleCI build $CIRCLE_BUILD_NUM"
+                --commit-sha       "$CIRCLE_SHA1"
+                --build-server-url "$CIRCLE_BUILD_URL"
+                --repository-url   "$CIRCLE_REPOSITORY_URL"
+```
+
+### Beta By Crashlytics
+
+1. Log in to Fabric.io and visit your organization's settings page.
+![](  {{ site.baseurl }}/assets/img/docs/fabric-org-settings-page.png)
+
+2. Click your organization (CircleCI in the image above), and click 
+the API key and Build Secret links to reveal the items.
+![](  {{ site.baseurl }}/assets/img/docs/fabric-api-creds-page.png)
+
+3. Navigate to your App's Project Settings page in the CircleCI app, and under
+Environment Variables add two new items named `CRASHLYTICS_API_KEY` and
+`CRASHLYTICS_SECRET`, with the values you find on Crashlytics website.
+
+4. Modify the deployment section of your `config.yml` as
+follows:
+
+```
+    steps:
+      - deploy:
+          name: crashlytics deploy
+          command: |
+            - gym
+            - ipa distribute:crashlytics
+                --crashlytics_path Crashlytics.framework
+                --api_token    "$CRASHLYTICS_API_KEY"
+                --build_secret "$CRASHLYTICS_SECRET"
+```
+
+### TestFairy
+
+To set up your app on TestFairy follow these steps:
+
+1. Visit the preferences page in the
+TestFairy dashboard and navigate to the API Key section. 
+2. Copy your API
+key and go to your App's Project settings on CircleCI. 
+3. Add a new
+Environment Variable named `TESTFAIRY_API_KEY` and paste in the API key
+from the TestFairy dashboard.
+
+4. Edit your `config.yml` as follows:
+
+```
+    steps:
+      - deploy:
+          name: testfairy deploy
+          command: |
+            - gym
+            - ipa distribute:testfairy
+                --key     "$TESTFAIRY_API_KEY"
+                --comment "CircleCI build $CIRCLE_BUILD_URL"
+```
+
+## Resolving Common Simulator Issues
+
+A series of simulator-related issues are known to happen on some
+projects. Here are the most frequent of those:
+
+* **Xcode version is not available.** We install
```

Comment:
Need to replace this example with `"8.3"` (quotes and higher number) instead of `7.3` so that this is still relevant on 2.0